introduction.md

```diff
@@ -1,2 +1,6 @@
 Hello you randos.
-I like Javascript and Java
\ No newline at end of file
+I like Javascript and Java
+*Crying
+*Eating
+*Sleeping
+*Walking
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?